### ISSUE TITLE
[Android] Pause event triggered when starting from notification

### ISSUE THREAD

**Issue by alexislg2** [phonegap/phonegap-plugin-push#647]
### Expected Behaviour
When I start the app from a notification I expect only the resume event to be triggered

### Actual Behaviour
When I start the app from a notification, resume event is triggered then pause event then resume event

### Reproduce Scenario (including but not limited to)
It only happens when app is in pause for at least 5 minutes.

#### Steps to Reproduce 
start the app. Pause the app. Wait 5 minutes. Send a GCM message to the device. Click on that notification

#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)
Android 6 + Cordova 5.1.1 + latest plugin version

#### Sample Push Data Payload
title and message but it happens with every kind of payload

#### Sample Code that illustrates the problem

    document.addEventListener("pause", function () {
        console.log("pause");
    }, false);
    document.addEventListener("resume", function () {
        console.log("resume");
    }, false);

#### Logs taken while reproducing problem

When starting from notification bar :
    resume
    pause
    resume

**Comment by macdonst** [comment]
@alexislg2 can you check to see if the same behaviour is present when you start the app but not from a push notification? I have a feeling this is a cordova-android issue. So try putting the app in the background, waiting 5 minutes and starting it from the app switcher instead of a push notification. If you have already tried this please let me know the result.

**Comment by alexislg2** [comment]
Yes I tried that. I comfirm that problem happens only when app resumed fron
status bar

**Comment by macdonst** [comment]
@alexislg2 awesome, thanks for the clarification.

**Comment by alexislg2** [comment]
The fact that it happens only after a given time is quite strange also

**Comment by macdonst** [comment]
@alexislg2 Android probably when PushHandlerActivity forces the main activity to start.

**Comment by MaxInMoon** [comment]
just an information:

for me this problem happened on android only at first app launch because of the alert box prompting  location authorization. It triggers a `pause` event when the box appears.
